query-engine/query-engine-node-api/src/engine.rs

```diff
@@ -45,6 +44,7 @@ enum Inner {
 /// Holding the information to reconnect the engine if needed.
 #[derive(Debug, Clone)]
 struct EngineDatamodel {
+    #[allow(unused)]
     ast: Datamodel,
```

Comment:
we could easily delete that and more, but I am not familiar enough with the node api glue code to say if that would break the TS code, so I abstained from going further here